lib/accelerometer.js

```diff
@@ -167,7 +167,7 @@ var Controllers = {
   MMA7361: {
     DEFAULTS: {
       value: {
-        zeroV: [336, 372, 287],
+        zeroV: [330, 350, 290],
```

Comment:
Sure. I did this because the calibration is never going to be perfect for all users. If we're going to pick values based on the first person's calibration (I'm assuming that's where these numbers came from), we might as well just round the values to nice looking numbers that represent an average across tested devices. An MMA7361 in an Esplora shouldn't really have different values than an MMA7361 in other settings. FWIW, the values that ship in `Esplora.cpp` are `[320,330,310]`.